guides/spin/cli/index.md

```diff
@@ -95,7 +95,7 @@ to see examples for acquiring a clientId/clientSecret from your provider.
 This OAuth2 configuration method needs to be initialized once to authenticate with the provider before
 it can be used for automation. To authenticate, configure OAuth2 as shown above and execute
 any `spin` command. You will be prompted to authenticate with your OAuth2 provider
-and paste an access code. `spin` then exchanges the code for an OAuth2 access/refresh token pair,
+and paste an access code. This process may involve configuring the `callback url` to be http://localhost:8085 in order to view and retrieve the access code to be provided to the prompt. `spin` then exchanges the code for an OAuth2 access/refresh token pair,
```

Comment:
@atanner27 
@jtk54 
"...may involve..." makes this a little vague. Can we predict whether this will be needed, or under what circumstances?

Reply:
This should probably be documented per oauth2 provider.